PCSX2/pcsx2
# wx: Convert GS Capture Settings window to wxWidgets

### Description of Changes


Part of the effort to convert non-wx windows to wx to support translations.

This also makes the dialog usable on linux (minus the codec support)

This is currently blocked in a few ways by the recent WIL changes:
- recordings always crash the emulator when being stopped (quartz.dll?)
- F12 to open the window also results in a crash

Windows:
![gsCaptureWindowWin32](https://user-images.githubusercontent.com/13153231/133907165-58e059a5-dd0a-4b27-a594-72cf86fa9007.png)

### Rationale behind Changes


Effort towards the 1.8 release.

### Suggested Testing Steps


- Open the dialog with F12, and via the Capture menu.  It should only open if a game is running
- Try to create a recording on linux and on windows, both with and without a codec (for windows)
- Try to modify the filepath by hand, and via browsing
- Try configuring the codec / changing the width-height / colorspace
